### ISSUE TITLE
Forms

### ISSUE THREAD

**Issue by StreiffBerenice** [Orange-OpenSource/Orange-Boosted-Bootstrap#164]
- [x] The state "checked" should have a suficient colour contrast (non conformity :  white on orange) 
- [x] Textual fields are grey on white solution ->use a darker grey 
- [x] Respect the ergonomics rules about the proximity between the textual fields and its labels.  These two elements should be closer.
- [x] The focus on the selection lists don't have a suficient colour contrast 
- [ ] All forms with required fields should have a kind of legend at the begening indicating " required fields are etc" 
- [ ] You don't put help with data entry (aide à la saisie, english traduction ?) as the data format (for the fields "textarea", put the number of authorized characters and for the "password" field, put the formt) to reduce the penibility when users fill in the forms.

**Comment by ffoodd** [comment]
FWIW, the four first items are defined in Orange Brand Digital Guidelines, thus will need additionnal work on their side before changing it in Boosted.

And the two last items would more require explicit examples in documentation website (and probably in [Orange Accessibility Guidelines](https://github.com/Orange-OpenSource/a11y-guidelines), like [accessible form example](https://a11y-guidelines.orange.com/web_EN/exemples/formulaire/index.html)).

**Comment by ffoodd** [comment]
The two last items will be handled in v5 new website, see #261.
